src/billing/components/BillingPageContents.tsx

```diff
@@ -4,13 +4,18 @@ import React, {FC} from 'react'
 // Components
 import BillingFree from 'src/billing/components/Free/Free'
 import BillingPayAsYouGo from 'src/billing/components/PayAsYouGo/PayAsYouGo'
+import MarketplaceBilling from 'src/billing/components/marketplace/MarketplaceBilling'
 
 // Utils
 import {useBilling} from 'src/billing/components/BillingPage'
 
 const BillingPageContents: FC = () => {
   const [{account}] = useBilling()
 
+  if (account.type === 'pay_as_you_go' && account.marketplace !== 'Zuora') {
```

Comment:
this will likely need to be `&& !!account.marketplace` because zuora is not a marketplace

Reply:
Good to know, thanks!

Reply:
nvm, answered my own question by searching the `yml`